### ISSUE TITLE
Unyson Page Builder Not Working

### ISSUE THREAD

**Issue by mattharms87** [ThemeFuse/Unyson#3660]
I'm running Kerge theme, and the Unyson page builder stopped working abruptly. I've seen similar issues on this forum and tried some initial fixes, but nothing seems to be working.

**Comment by ImprobableCode** [comment]
What WordPress version are you running ? can you do anything? more details required

**Comment by lmpixelsteam** [comment]
> I think I figured out the issue. Looks like it Unyson is not yet compatible with Wordpress 5.0. Will try to install a plugin to go back to the classic editor.

Yes, we have provided this solution to you: "Please install and activate this plugin - https://wordpress.org/plugins/classic-editor/ , then you can use the Unyson Visual Page Builder. Thanks."

**Comment by danyj** [comment]
Add this in functions.php and  dont worry about GB for now until this https://github.com/ThemeFuse/Unyson-PageBuilder-Extension/pull/110 is out 


```
/*
 * Disable block editor
 * in case you need some logic see:
 * https://gist.github.com/danyj/ec00057550fd6f73995ab2f8fc8b729f
 */
if (!function_exists('_thz_filter_disable_block_editor_pt')){
	function _thz_filter_disable_block_editor_pt( $use_block_editor, $post_type ){
	  
	  $use_block_editor = false;
	  return  $use_block_editor;
	  
	}
}

add_filter( 'use_block_editor_for_post_type', '_thz_filter_disable_block_editor_pt', 10, 2 );
```

**Comment by anandspybs** [comment]
I had the same problem, but I have fixed it. You need to do:
1. Deactivate the plugin (Unyson)
2. Delete the plugin
3. Delete the cache
4. Install again. 
Now that would be working fine.
